projects/nfstream/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+python3 -m pip install -U -r dev_requirements.txt
+python3 prepare.py
+python3 -m pip install -U  .
+
+for fuzzer in $(find $SRC -name '*_fuzzer.py'); do
```

Comment:
did you try using `compile_python_fuzzer` in the body of this loop? This will enable you to get code coverage visualisation as well. You can parse flags to `pyinstaller` using this script as well, see e.g. https://github.com/google/oss-fuzz/blob/d412c6b08b0eea1e1def2bd0b19f40b5da6b0fa5/projects/django/build.sh#L26